src/main/java/com/library/constant/ApplicationConstant.java

```diff
@@ -1,6 +1,8 @@
 package com.library.constant;
 
 
+import java.time.format.DateTimeFormatter;
+
 public class ApplicationConstant {
```

Comment:
What is the difference between Class, Interface and Abstract Class?
Why class is used hear?

Reply:
https://www.baeldung.com/java-constants-good-practices

Reply:
What will be next time? 700+ pages book?

Any question need short clear answer written by hands.

And why class still is defined here?

Reply:
Because using interface for constatnt class is antipattern
Abstract class is used when need to provide common logic for childs classes
If question is about "why class isnt final?", that is my mistake